spec/index.html

```diff
@@ -299,6 +299,66 @@ <h3>Object Lists</h3>
     Turtle provides a number of ways of writing each.
   </p>
 
+  <section id="sec-version">
+    <h3>Version Announcement</h3>
+
+    <p>The Turtle language has evolved since its origin, and RDF 1.2
+      introduces new features which may confuse non-compliant parsers.
+      RDF 1.2 Turtle introduces the
+      <a href="#grammar-production-sparqlVersion"><code>VERSION</code></a> and
+      <a href="#grammar-production-version"><code>@version</code></a> directives
+      along with an optional `version` <a href="#sec-mediaReg">HTTP Media Type parameter</a>.
+      When serializing Turtle with new features such as base direction or triple terms,
+      authors can announce the presense of these features using these directives.
+    </p>
+
+    <pre id="ex-version-decl"
+         class="example turtle" data-transform="updateExample"
+         title="Version announcement">
+      <!--
+      @version "1.2" .
+      @prefix : <http://example.com/> .
+      :a :name "Alice" ~ :t {| :statedBy :bob ; :recorded "2021-07-07"^^xsd:date |} .
+      -->
+    </pre>
+
+    <p>Alternatively, the SPARQL style directive can be used
+      (note that there is no trailing <a href="#cp-full-stop"><code title="full stop">.</code></a>):</p>
+
+    <pre id="ex-sparql-version-decl"
+         class="example turtle" data-transform="updateExample"
+         title="SPARQL-style version announcement">
+      <!--
+      VERSION "1.2"
+      PREFIX : <http://example.com/>
+      :a :name "Alice" ~ :t {| :statedBy :bob ; :recorded "2021-07-07"^^xsd:date |} .
+      -->
+    </pre>
+
+    <p>When providing content over HTTP, servers can announce the version
+      using the optional `version` Content-Type parameter:</p>
+
+    <pre id="ex-http-version-decl"
+         class="example turtle" data-transform="updateExample"
+         title="HTTP version announcement">
+      <!--
+      GET /document.ttl HTTP/1.1
+      Host: example.com
+      Accept: text/turtle; version=1.2
+      -->
+    </pre>
+
+    <p>In the absense of an explicit version announcement,
+      processors should treat the document as having version "1.2".</p>
+
+    <p class="note">Version numbers other than "1.2" are not defined by
+      this specification.
+      Processors may treat this as an error or warning.
+      While a version of "1.1" is not defined, processors may treat this
+      as an indication that no RDF 1.2 features are used, so that
+      if they are encountered when parsing a document, it can be treated as an error.</p>
```

Comment:
I'm not sure we need this part. I feel it could introduce more confusion than needed.
Publishers may interpret this as being allowed to specify version 1.1, while this will break older 1.1 parsers.